Assets/Scripts/Enemy/Knockback.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Knockback : MonoBehaviour
+{
+    [SerializeField]
+    private Rigidbody2D rb2d;
+
+    [SerializeField]
+    private float strength = 16f, delay = 0.15f;
```

Comment:
I had to set this to a fairly large value (~10,000) to get a sizable knockback. Might be worth playing with the delay too.